frontend/sozisel-app/src/model/utils/session.ts

```diff
@@ -0,0 +1,7 @@
+import { Session } from "../../graphql";
+
+export default function getSessionStatus(session: Session): string {
```

Comment:
You could transfer that to hook so you can use translations. It will actually make a lot of sense as you call this function on the same sessions several times and with a hook you will get a single translation string.